openedx/core/djangoapps/credit/signals.py

```diff
@@ -70,3 +75,46 @@ def listen_for_grade_calculation(sender, username, grade_summary, course_key, de
                     api.set_credit_requirement_status(
                         username, course_id, 'grade', 'grade', status="failed", reason={}
                     )
+
+
+def tag_course_content_with_partition_scheme(course_key, partition_scheme):
+    """ Create user partitions with provided partition scheme and tag credit
+    blocks for the given course with those user partitions.
+
+    Args:
+        course_key (CourseKey): Identifier for the course.
+        partition_scheme (str): Name of the user partition scheme
+
+    """
+    partition = None
+    # create partition with provided partition scheme
+    if partition_scheme == 'verification':
+        partition = VerificationPartitionScheme()
+
+    if partition is None:
+        log.error(u'No user partition found with scheme "%s".', partition_scheme)
+        # TODO: maybe raise a 'NoSuchUserPartitionError' exception if not user partition is present with provide 'partition_scheme'.
```

Comment:
@wedaly Do we need to raise exception `NoSuchUserPartitionError` if provided `partition_scheme` is not present?

Also I tried to make the method `tag_course_content_with_partition_scheme` generic so that we can tag content with different parition_scheme (although we will be tagging course content with just `NoSuchUserPartitionError`).
Do we need such a general method or just tag content with hardcoded `VerificationPartitionScheme`?